course.py

```diff
@@ -144,7 +145,10 @@ def course_info(course_id, year):
     all_years = db.session.query(Course).filter_by(id=course.id).distinct(Course.year).order_by(
         Course.year.desc()).all()
 
-    return render_template('course_info.html', course=course, all_years=all_years, current_year=year)
+    evaluations = db.session.query(Evaluation).filter_by(course_id=course_id).first()
```

Comment:
```suggestion
    evaluations = db.session.query(Evaluation).filter_by(course_id=course_id).all()
```

or something similar, I guess...